doc/source/timeseries.rst

```diff
@@ -900,12 +900,31 @@ calendar time arithmetic. :class:`CalendarDay` is useful preserving calendar day
 semantics with date times with have day light savings transitions, i.e. :class:`CalendarDay`
 will preserve the hour before the day light savings transition.
 
+Addition with :class:`CalendarDay`:
+
 .. ipython:: python
 
    ts = pd.Timestamp('2016-10-30 00:00:00', tz='Europe/Helsinki')
    ts + pd.offsets.Day(1)
    ts + pd.offsets.CalendarDay(1)
 
+Creating a :func:`date_range`:
+
+.. ipython:: python
+
+   start = pd.Timestamp('2016-10-30 00:00:00', tz='Europe/Helsinki')
+   pd.date_range(start, freq='D', periods=3)
+   pd.date_range(start, freq='CD', periods=3)
+
+Resampling a timeseries:
+
+.. ipython:: python
+
+   idx = pd.date_range("2016-10-30", freq='H', periods=4*24, tz='Europe/Helsinki')
+   s = pd.Series(range(len(idx)), index=idx)
+   s.resample('D').count()
+   s.resample('CD').count()
```

Comment:
Is this supposed to work on master? After a clean build locally I get a `ValueError: Invalid frequency: CD`. 

Don't think doctest covers these files; if it does then my mistake but figured it was worth asking

Reply:
This currently works on master:

```
In [3]: pd.__version__
Out[3]: '0.24.0.dev0+563.g0976e1261'

In [4]:    idx = pd.date_range("2016-10-30", freq='H', periods=4*24, tz='Europe/
   ...: Helsinki')
   ...:    s = pd.Series(range(len(idx)), index=idx)
   ...:    s.resample('D').count()
   ...:    s.resample('CD').count()
   ...:
   ...:
Out[4]:
2016-10-30 00:00:00+03:00    25
2016-10-31 00:00:00+02:00    24
2016-11-01 00:00:00+02:00    24
2016-11-02 00:00:00+02:00    23
Freq: CD, dtype: int64
```

I am not sure if the doctest uses master or the the latest tagged branch.